po/openpetfoodfacts/openpetfoodfacts.pot

```diff
@@ -36,3 +36,30 @@ msgstr ""
 msgctxt "android_app_link"
 msgid "https://play.google.com/store/apps/details?id=org.openpetfoodfacts.scanner&hl=en"
 msgstr ""
+
+msgctxt "brands_example"
+msgid "Purina, Pedigree"
+msgstr ""
+
+msgctxt "product_name_example"
+msgid "Pro Plan Adult, rich in chicken"
+msgstr ""
+
+msgctxt "generic_name_example"
+msgid "Complete pet food for adult cats"
+msgstr ""
+
+msgctxt "categories_example"
+msgid "Dry adult cat food"
+msgstr ""
+
+msgctxt "labels_example"
+msgid "No sugar, No colorings"
+msgstr ""
+
+
```

Comment:
removing extra line

Reply:
Are you removing it, or did you want to say I should?

Reply:
suggestion to remove before I merge